query-languages/m/m-spec-consolidated-grammar.md

```diff
@@ -564,13 +564,19 @@ _error-raising-expression:_<br/>
 #### Error handling expression
 
 _error-handling-expression:_<br/>
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`try` _protected-expression otherwise-clause<sub>opt</sub><br/>
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`try` _protected-expression 
+error-handler<sub>opt</sub><br/>
 protected-expression:<br/>
 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;expression<br/>
+_error-handler:_<br/>
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;otherwise-clause<br/>
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;catch-function<br/>
 otherwise-clause:_<br/>
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`otherwise` _default-expression<br/>
-default-expression:<br/>
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;expression_
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`otherwise` _default-expression_<br/>
+_default-expression_:<br/>
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;_expression_<br/>
+_catch-function:_<br/>
```

Comment:
Should this be called "catch-clause"? Also, it's missing the "catch" keyword.

Reply:
Ah. Good, umm, catch. :-D

Added "catch". Following the pattern of "_otherwise-clause_: `otherwise` _default-expression_", added "_catch-clause_: `catch` _catch-function_"